### ISSUE TITLE
ConstantName and Structs

### ISSUE THREAD

**Issue by jurriaan** [rubocop/rubocop#432]
``` ruby
Person = Struct.new(:first_name, :last_name) do

end
```

gives this offence: Use SCREAMING_SNAKE_CASE for constants.

but,

``` ruby
Person = Struct.new(:first_name, :last_name)
```

does not.

Both are not constant definitions, so the 
ConstantName offence shouldn't be triggered.
